gdk4/src/dmabuf_texture_builder.rs

```diff
@@ -0,0 +1,66 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::translate::*;
+
+use crate::{DmabufTextureBuilder, Texture};
+
+impl DmabufTextureBuilder {
+    #[doc(alias = "gdk_dmabuf_texture_builder_build")]
+    #[must_use = "The builder must be built to be used"]
+    #[allow(clippy::missing_safety_doc)]
+    pub unsafe fn build(self) -> Result<Texture, glib::Error> {
+        let mut error = std::ptr::null_mut();
+
+        let result = ffi::gdk_dmabuf_texture_builder_build(
+            self.to_glib_none().0,
+            None,
+            std::ptr::null_mut(),
```

Comment:
You want to do the same as with the GL texture builder here so that an optional `FnOnce() + Send + 'static` closure can be provided here

Reply:
Oh I should've scrolled down, sorry :)